src/main/java/pl/wedding/management/service/reservation/domain/converter/ReservationConverter.java

```diff
@@ -11,13 +12,13 @@
 public class ReservationConverter implements Converter<ReservationDto, ReservationEntity> {
 
     @Override
-    public ReservationDto convertFromEntity(ReservationEntity reservationEntity) {
-        return new ReservationDto(reservationEntity.getId(),
-                reservationEntity.getReservationStart().truncatedTo(ChronoUnit.MINUTES),
-                reservationEntity.getReservationEnd().truncatedTo(ChronoUnit.MINUTES),
-                reservationEntity.getWeddingHall(),
-                reservationEntity.getTenant(),
-                reservationEntity.getNumberOfGuests(),
-                reservationEntity.getOccasion());
+    public ReservationDto convertFromEntity(@NonNull ReservationEntity entity) {
+        return new ReservationDto(entity.getId() == null ? null : entity.getId(),
```

Comment:
Czy to porównanie ma sens? Przeczytajmy to na głos: weźmy ID z encji. Jeśli jest null, zwróćmy null, w innym przypadku, zwróćmy jego wartość.
Pozwól, że zapiszę to prościej: `entity.getId()`

Reply:
Ujednoliciłem gdy zmieniałem typy, ale faktycznie tak będzie czytelniej.

Reply:
Domyślam się, że Bartek źle zrozumiał nasze poprzednie uwagi dotyczące lecących NPE. 

Bartek, nie każdą wartość musisz przed tym zabezpieczać. :) Chodzi o to, że wołałeś metodę `.trucatedTo(ChronoUnit)` na wartości zwracanej z `getReservationEnd()`. W momencie kiedy ta wartość byłaby nieustawiona to wołałbyś coś na zasadzie `null.truncatedTo(...)` co rzuciłoby wyjątkiem. Nie musisz się przed tym zabezpieczać przy każdej wartości.  
Chciałbym Cię też prosić, żebyś zamiast wykorzystywania == ? : użył Optionala i wyeksportował tą logikę do metody.

Reply:
ok, puściłem już jednego commita przed Twoimi uwagami - jutro zrobię tego optionala.